packages/vitest/src/typecheck/typechecker.ts

```diff
@@ -310,15 +310,17 @@ export class Typechecker {
     }
     this._output = ''
     this._startTime = performance.now()
-    const child = execa(typecheck.checker, args, {
-      cwd: root,
-      stdout: 'pipe',
-      reject: false,
+    const child = x(typecheck.checker, args, {
+      nodeOptions: {
+        cwd: root,
+        stdio: 'pipe',
```

Comment:
`stdio: 'pipe'` is equivalent to `stdio: ['pipe', 'pipe', 'pipe']` which is the default in `execa`.